message_processor.go

```diff
@@ -111,7 +109,6 @@ func (mg messageProcessor) sendMessage(l logger.Logger, buf *logdriver.LogEntry,
 	if !buf.Partial || t.shouldFlush(time.Now()) {
 		msg.Line = t.tBuf.Bytes()
 		msg.Source = buf.Source
-		msg.Partial = buf.Partial
```

Comment:
any negative impact of removing this?

Reply:
This will break partial log functionality